core/contracts/tokenized-derivative/TokenizedDerivativeCreator.sol

```diff
@@ -2,9 +2,9 @@ pragma solidity ^0.5.0;
 
 pragma experimental ABIEncoderV2;
 
-import "../common/implementation/Testable.sol";
-import "./AddressWhitelist.sol";
+import "../common/AddressWhitelist.sol";
```

Comment:
Why is this and the AddressWhitelist move in this PR?

Reply:
this was a mistake from github not including a commit and a mix up between the previous pr #961 . Thanks for the catch, have resolved.